main.go

```diff
@@ -981,6 +1013,30 @@ var booleanFlags = map[string]bool{
 	"-benchmem": true,
 }
 
+func filterBuildFlags(flags []string) (filtered []string) {
+	for i := 0; i < len(flags); i++ {
```

Comment:
`for i, arg := range flags`

Reply:
We can't do that, because we do an `i++` inside the loop.

Reply:
@mvdan still works (I think)

Reply:
It would be a different loop altogether. In a range, you can't skip an element, which we do here.